docker-compose.dev.yml

```diff
@@ -19,26 +19,33 @@ services:
   validator:
     image: hyperledger/sawtooth-validator:1.0.1
     depends_on:
-      - remme-genesis
       - settings-tp
+    volumes:
+      - validator_keys:/etc/sawtooth/keys
     expose:
       - 4004
-    volumes:
-      - user_keys:/root/.sawtooth/keys
-      - genesis_data:/genesis/batch
+    ports:
+      - '${REMME_VALIDATOR_PORT:-8800}:8800'
+    env_file:
+      - .env
     entrypoint: |
       bash -c "
-        sawadm keygen && \
-        sawtooth keygen key
-        if [ -d "/var/lib/sawtooth" ]; then
-          rm /var/lib/sawtooth/*
+        if [ ! -f /etc/sawtooth/keys/validator.priv ]; then
+          sawadm keygen
+        fi
+        ADDITIONAL_ARGS=\"--peering $$REMME_PEERING_MODE\"
+        if [ ! -z \"$$REMME_PEERS_LIST\" ]; then
```

Comment:
is this readable ADDITIONAL_ARGS  if statement masterpiece really needed?
Can we get aways with just
--seeds $$REMME_PEERS_LIST
--peers $$REMME_PEERS_LIST
in the entrypoint?

Reply:
Will fix it